### PR TITLE
BEL-3216 Switch to Alb component built by Caylent to remove listener on port 3000

### PR DESCRIPTION
**Pull request by tpaulshippy**
[Link to Jira ticket](https://strongmind.atlassian.net/browse/BEL-3216)

## Purpose 

The way we were using the awsx component to create the load balancer caused a listener on port 3000 to get created which got hit by bots and resulted in 5xx errors on our load balancer that alarmed.

## Approach 

Instead of using the pulumi.awsx component to create the load balancer, we are switching to use the component created by the Caylent team. This removes the "default target group" that created a listener pointed at port 3000.

## Testing

We tested this change by rolling it out to multiple projects including frozen desserts, repository dashboard, and canvas-alerts-api-rails. 

## Screenshots/Video

Before

image

After

image
image

### CODE REVIEW

File: deployment/src/strongmind_deployment/container.py
Comment (markng) [inline deployment/src/strongmind_deployment/container.py:442]: If it was like this before, my comments above are likely not something to worry about.

File: deployment/src/strongmind_deployment/container.py
Comment (markng) [inline deployment/src/strongmind_deployment/container.py:528]: Did we check that this worked? If so, great!

**Review by markng — APPROVED** [review]

File: deployment/src/strongmind_deployment/alb.py
Comment (tpaulshippy) [reply deployment/src/strongmind_deployment/alb.py:116]: I think you might be right here. I don't think these logs were on in identity stage.

File: deployment/src/strongmind_deployment/container.py
Comment (tpaulshippy) [reply deployment/src/strongmind_deployment/container.py:528]: Yes this fixed the issue that caused me to take down repo dashboard.

File: deployment/src/strongmind_deployment/alb.py
Comment (tpaulshippy) [reply deployment/src/strongmind_deployment/alb.py:118]: Parameterized the account ID.

**Comment by tpaulshippy** [comment]
Need to wait until Thursday night to merge this.

**Pull request merged**
